reframe/frontend/executors/__init__.py

```diff
@@ -415,18 +437,21 @@ def runall(self, testcases, restored_cases=None):
             # Print the summary line
             num_failures = len(self._stats.failed())
             num_completed = len(self._stats.completed())
+            num_skipped = len(self._stats.skipped())
             num_tasks = len(self._stats.tasks())
-            if num_failures > 0 or num_completed < num_tasks:
+            if num_failures > 0 or num_completed + num_skipped < num_tasks:
                 status = 'FAILED'
             else:
                 status = 'PASSED'
 
             total_run = len(testcases) - num_tasks + num_completed
+            total_completed = len(self._stats.completed(0))
+            total_skipped = len(self._stats.skipped(0))
```

Comment:
Previously, we were reporting as the total number of completed tasks only those from the last run, which is wrong when we have retries. The same would be for skipped tests. It would report always 0 when using retries, because if you skip a test, it is not retried. So I changed these numbers to always take the values of the first run.

Reply:
Ok, we can calculate how many were completed by adding all the runs (in case some were succefsful in a later try). And the same for the skipped, since skipped will not be rerun the sum should be fine. Also I think the `abort_reason = None` in line 403/425 is forgotten in the code and is not actually used.

Reply:
Sorry, completed is succeded or failed so it's not correct. But then maybe we have to reconsider what we print in this line: `f'Ran {total_completed}/{total_run}'`. `total_run` is the tests that finished (failed or succeeded, right?) but not counting the aborted. The total completed sounds to me wrong to be only the first run